app/helpers/followables_helper.rb

```diff
@@ -20,7 +20,7 @@ def render_follow(follow)
   end
 
   def followable_class_name(followable)
-    followable.class.to_s.parameterize.gsub('-', '_')
+    followable.class.to_s.parameterize('_')
```

Comment:
I didn't knew about this :) but `parameterize` accepts a param for the substitution http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-parameterize